homeassistant/components/demo/sensor.py

```diff
@@ -11,16 +14,20 @@
     ATTR_BATTERY_LEVEL,
     CONCENTRATION_PARTS_PER_MILLION,
     ENERGY_KILO_WATT_HOUR,
+    ENERGY_MEGA_WATT_HOUR,
     PERCENTAGE,
     POWER_WATT,
     TEMP_CELSIUS,
+    VOLUME_CUBIC_FEET,
+    VOLUME_CUBIC_METERS,
 )
-from homeassistant.core import HomeAssistant
+from homeassistant.core import Event, HomeAssistant, callback
 from homeassistant.helpers.entity import DeviceInfo
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType, StateType
 
 from . import DOMAIN
+from ...helpers.event import async_track_time_interval
```

Comment:
I think we try to avoid having relative imports in this case.
```suggestion
from homeassistant.helpers.event import async_track_time_interval
```